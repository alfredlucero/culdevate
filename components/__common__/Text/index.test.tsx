import React from "react";
import { render, cleanup } from "react-testing-library";
import { H1, H2, H3, P } from "./index";

afterEach(cleanup);

describe("Text (H1, H2, H3, P)", () => {
  test("should render all Text components", () => {
    const { container } = render(
      <div>
        <H1>Culdevate Header 1 Text</H1>
        <H2>Culdevate Header 2 Text</H2>
        <H3>Culdevate Header 3 Text</H3>
        <P>Culdevate Paragraph Text</P>
        <P small>Culdevate Paragraph Small Text</P>
      </div>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
