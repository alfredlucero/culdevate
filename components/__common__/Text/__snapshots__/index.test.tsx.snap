// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text (H1, H2, H3, P) should render all Text components 1`] = `
<div>
  <h1
    class="sc-bdVaJa bVbrxZ"
  >
    Culdevate Header 1 Text
  </h1>
  <h2
    class="sc-bwzfXH jdFXXL"
  >
    Culdevate Header 2 Text
  </h2>
  <h3
    class="sc-htpNat iTJEtc"
  >
    Culdevate Header 3 Text
  </h3>
  <p
    class="sc-bxivhb fftNLG"
  >
    Culdevate Paragraph Text
  </p>
  <p
    class="sc-bxivhb dIdlXt"
  >
    Culdevate Paragraph Small Text
  </p>
</div>
`;
